plugins/guests/arch/cap/nfs_client.rb

```diff
@@ -0,0 +1,16 @@
+module VagrantPlugins
+  module GuestArch
+    module Cap
+      class NFSClient
+        def self.nfs_client_install(machine)
+          comm = machine.communicate
+          comm.sudo <<-EOH.gsub(/^ {12}/, '')
+            pacman -Sy --noconfirm
+            pacman -S --noconfirm nfs-utils
```

Comment:
Anyway, in one or two lines, the PR is pefect for me, please @mitchellh aprove it. :smile: